### DISCUSSION TITLE
How can I redirect url?

### DISCUSSION THREAD

**Discussion by Toaster496** [n3d1117/chatgpt-telegram-bot#402] [Q&A]
I have my own self hosted GPT and I have a api url end point. So i was worndering if i can redicect the url to my GPT but keep the web server

**Comment by johe123qwe** [comment]
/lib64/python3.8/site-packages/openai/__init__.py
`api_base = os.environ.get("OPENAI_API_BASE", "https://api.openai.com/v1")`
You can modify the api url in python's openai module
